src/dredd-command.coffee

```diff
@@ -8,6 +8,7 @@ spawnArgs = require 'spawn-args'
 
 Dredd = require './dredd'
 interactiveConfig = require './interactive-config'
+{applyLoggingOptions} = require './apply-configuration'
```

Comment:
I am not sure with the filename `apply-configuration`, when it expose more then one function. Maybe `configuration` only would be enough?